test/helpers/kubectl.go

```diff
@@ -635,21 +635,30 @@ func (kub *Kubectl) labelNodes() error {
 		return fmt.Errorf("unable to unmarshal string slice '%#v': %s", nodesList, err)
 	}
 
-	index := 1
+	var (
+		index int = 1
+
+		noCiliumNode     = GetNodeWithoutCilium()
+		noCiliumNodeName string
+	)
 	for _, nodeName := range nodesList {
```

Comment:
What's the guarantee that `nodesList` will be ordered with `k8s3` as the last node? Can't `kubectl get nodes -o json` return the nodes in any order?

Reply:
Good question. The ordering seems to be oldest node to newest node. Since we create k8s1 first and then the rest, that assumption holds true. I guess this code has existed since the beginning, and hasn't proven to be a problem...yet :)

Reply:
I guess we could enforce that by using `--sort-by`